client/factory.go

```diff
@@ -216,6 +216,11 @@ func (b *clientFactory) createTLSConfig(c *cli.Context) (*tls.Config, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to read TLS disable host verification flag: %w", err)
 	}
+	enableTLSS := c.String(common.FlagTLS)
+	enableTLS, err := strconv.ParseBool(enableTLSS)
```

Comment:
what's the reason to do custom parsing? (instead of `c.Bool(..)`)

Reply:
I'm doing custom parsing to keep consistency with your code above: https://github.com/temporalio/cli/blob/6a16f1c6ef215dfe88b4bb8598bd7c21bc4976c9/client/factory.go#L214

Reply:
ok so these values are accepted with strconv.ParseBool, true: `"1", "t", "T", "true", "TRUE", "True"`, false: `"0", "f", "F", "false", "FALSE", "False"`

I'd go with the c.Bool for consistency with the rest of the CLI

Reply:
updated